mixcoatl/auth.py

```diff
@@ -2,29 +2,28 @@
 import base64
 import hashlib
 import hmac
-from mixcoatl.settings.load_settings import settings
 
 
-def get_sig(http_method, path):
+def get_sig(http_method, config, path):
```

Comment:
I think this will break backward compatibility.  Can we make this config=None and default to the global object?